src/core/agent/services/identifierService.ts

```diff
@@ -147,6 +146,15 @@ class IdentifierService extends AgentService {
     }
   }
 
+  //Update multisig's status
+  async checkMultisigComplete(identifier: string): Promise<boolean> {
+    const metadata = await this.getMetadataById(identifier);
+    const markMultisigResult = await this.markMultisigCompleteIfReady(metadata);
+    return typeof markMultisigResult?.done == "boolean"
+      ? markMultisigResult.done
+      : true;
+  }
```

Comment:
Is it not just `return (await this.markMultisigCompleteIfReady(metadata)).done`?

Should always return `{ done: true|false }`

Reply:
This is not what I meant @Sotatek-HocNguyena - don't change the return type of this function.

I am saying `markMultisigResult` should always equal `{ done: true|false }`

so here we can just do `return (await this.markMultisigCompleteIfReady(metadata)).done`

Reply:
Hi @iFergal I've changed it.